addons/mail/static/src/models/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -0,0 +1,82 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one2one } from '@mail/model/model_field';
+import { clear, decrement, increment, insert, insertAndReplace, link, replace, unlink } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'mail.mobile_messaging_navbar',
+    identifyingFields: [['discuss', 'messagingMenu']],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeActiveTabId() {
+            if (this.discuss) {
+                return this.discuss.activeMobileNavbarTabId
+            }
+            if (this.messagingMenu) {
+                return this.messagingMenu.activeTabId;
+            }
+            return clear();
+        },
```

Comment:
Shouldn't the active tab be stored into this new model directly?

Reply:
Problem with `messagingMenu.activeTabId` that is also used in desktop.